table/block_based/block_based_table_factory.cc

```diff
@@ -310,6 +309,9 @@ static std::unordered_map<std::string, OptionTypeInfo>
          {offsetof(struct BlockBasedTableOptions, optimize_filters_for_memory),
           OptionType::kBoolean, OptionVerificationType::kNormal,
           OptionTypeFlags::kNone}},
+        // "use_delta_encoding" has not been persisted -
```

Comment:
nit: can you add a TODO marker for this message, so that it's clear this need some follow up.